modular_skyrat/modules/altborgs/code/robot_model.dm

```diff
@@ -6,12 +6,253 @@
+		"Robot" = list(SKIN_ICON_STATE = "robot_old", SKIN_ICON = R_ICON_STANDARD, SKIN_FEATURES = list(R_TRAIT_UNIQUEWRECK)),
+		"Bootyborg" = list(SKIN_ICON_STATE = "bootysd", SKIN_ICON = R_ICON_STANDARD),
+		"Male Bootyborg" = list(SKIN_ICON_STATE = "male_bootysd", SKIN_ICON = R_ICON_STANDARD),
+		"Protectron" = list(SKIN_ICON_STATE = "protectron_standard", SKIN_ICON = R_ICON_STANDARD),
+		"Miss M" = list(SKIN_ICON_STATE = "missm_sd", SKIN_ICON = R_ICON_STANDARD),
+		"Partyhound" = list(SKIN_ICON_STATE = "k69", SKIN_ICON = R_ICON_SERVICE_WIDE, SKIN_FEATURES = list(R_TRAIT_UNIQUEWRECK, R_TRAIT_WIDE)) //SLEEPER_OVERLAY = "k9sleeper"
+	)
+
 //SERVICE
 /obj/item/robot_model/service/skyrat
 	name = "Skyrat Service"
 	special_light_key = null
+	borg_skins = list(
+		"Waitress" = list(SKIN_ICON_STATE = "service_f", SKIN_LIGHT_KEY = "service"),
+		"Butler" = list(SKIN_ICON_STATE = "service_m", SKIN_LIGHT_KEY = "service"),
+		"Bro" = list(SKIN_ICON_STATE = "brobot", SKIN_LIGHT_KEY = "service"),
+		"Can" = list(SKIN_ICON_STATE = "kent", SKIN_LIGHT_KEY = "medical", SKIN_HAT_OFFSET = 3),
+		"Tophat" = list(SKIN_ICON_STATE = "tophat", SKIN_HAT_OFFSET = INFINITY),
+		"Sleek" = list(SKIN_ICON_STATE = "sleekserv", SKIN_ICON = R_ICON_SERVICE),
+		"Heavy" = list(SKIN_ICON_STATE = "heavyserv", SKIN_ICON = R_ICON_SERVICE),
+		"Bootyborg" = list(SKIN_ICON_STATE = "bootyservice", SKIN_ICON = R_ICON_SERVICE),
+		"Male Bootyborg" = list(SKIN_ICON_STATE = "male_bootyservice", SKIN_ICON = R_ICON_SERVICE),
+		"Protectron" = list(SKIN_ICON_STATE = "protectron_service", SKIN_ICON = R_ICON_SERVICE),
+		"Miss M" = list(SKIN_ICON_STATE = "missm_service", SKIN_ICON = R_ICON_SERVICE),
+		"Zoomba" = list(SKIN_ICON_STATE = "zoomba_green", SKIN_ICON = R_ICON_SERVICE, SKIN_FEATURES = list(model_features = R_TRAIT_UNIQUEWRECK, R_TRAIT_SMALL)),
+		"Mech" = list(SKIN_ICON_STATE = "lloyd", SKIN_ICON = R_ICON_SERVICE),
+		"Handy" = list(SKIN_ICON_STATE = "handy-service", SKIN_ICON = R_ICON_SERVICE),
+		"Darkhound" = list(SKIN_ICON_STATE = "k50", SKIN_ICON = R_ICON_SERVICE_WIDE, SKIN_FEATURES = list(R_TRAIT_UNIQUEWRECK, R_TRAIT_WIDE)), //SLEEPER_OVERLAY = "ksleeper"
+		"Vale" = list(SKIN_ICON_STATE = "valeserv", SKIN_ICON = R_ICON_SERVICE_WIDE, SKIN_FEATURES = list(R_TRAIT_UNIQUEWRECK, R_TRAIT_WIDE)), //SLEEPER_OVERLAY = "valeservsleeper"
+		"ValeDark" = list(SKIN_ICON_STATE = "valeservdark", SKIN_ICON = R_ICON_SERVICE_WIDE, SKIN_FEATURES = list(R_TRAIT_UNIQUEWRECK, R_TRAIT_WIDE)), //SLEEPER_OVERLAY = "valeservsleeper"
+		"Partyhound" = list(SKIN_ICON_STATE = "k69", SKIN_ICON = R_ICON_SERVICE_WIDE, SKIN_FEATURES = list(R_TRAIT_UNIQUEWRECK, R_TRAIT_WIDE)), //SLEEPER_OVERLAY = "k9sleeper"
+		"Borgi" = list(SKIN_ICON_STATE = "borgi-serv", SKIN_ICON = R_ICON_SERVICE_WIDE, SKIN_FEATURES = list(R_TRAIT_UNIQUEWRECK, R_TRAIT_WIDE, R_TRAIT_SMALL)), //SLEEPER_OVERLAY = "borgi-sleeper"
+	)
```

Comment:
Remove the sleeper_overlay part, we don't use it.

Reply:
guh, okay